browsersync.config.js

```diff
@@ -41,10 +41,17 @@ const localConfig = moduleExists( './local-config.json' )
  */
 module.exports = {
 	debugInfo: true,
+	files: [
+		'./wp-content/themes/core/dist/**/*.js',
+		'./wp-content/themes/core/dist/**/*.css',
+		'./wp-content/themes/core/**/*.php',
+		'./wp-content/plugins/core/**/*.php',
+	],
```

Comment:
I wonder if this can be consolidated to:
```
		`${ pkg.config.coreThemeDir }/dist/**/*.{js,css}`,
		`${ pkg.config.coreThemeDir }/**/*.{html,php}`, / <-- Added html, since some of our patterns and templates may be html-only.
		`${ pkg.config.corePluginDir }/**/*.php`, // <-- Needs to be added to package.json
```

Note that you'd have to require the package.json file at the top of this file, too (since it's not already included):
`const pkg = require( './package.json' );`